prisma/schema.prisma

```diff
@@ -30,33 +30,39 @@ model Claim {
   subject       String
   claim         String
   object        String?
-  qualifier     String?
+  statement     String?
   aspect        String?
-  howKnown      String?
-  source        String?
+  sourceUrl     String?
   effectiveDate DateTime?
   confidence    Float?
-  reviewRating  Int?
-  createdByUser User          @relation(fields: [userId], references: [id])
-  userId        Int
```

Comment:
`userId` is needed for the `createdByUser`. It is reference to the relation it has with the `User` model. You can see `userId` is referenced in `relation(fields: [userId], references: [id])`. You can change the name of the field `userId`. But, a field that will contain `User` model's id is needed. That's how Prisma works.